android/src/main/java/com/guichaguri/trackplayer/module/MusicEvents.java

```diff
@@ -36,6 +36,11 @@
     public static final String PLAYBACK_ERROR = "playback-error";
     public static final String PLAYBACK_UNBIND = "playback-unbind";
 
+
+    // ICY METADATA Events
+    public static final String ICY_HEADER_UPDATE = "icy-header-update";
+    public static final String ICY_MATADATA_UPDATE = "icy-metadata-update";
```

Comment:
Are those two events necessary? I mean, they could be implemented as one, such as `icy-update`.

Reply:
It could be implemented through `icy-update`, but as` icy-metadata-update` is received several times, and `icy-header-update` only in connection, to avoid a further check, I decided to separate in this way.

Reply:
You can add a property to tell which type of data you're receiving, but I don't think having two different events accomplish anything.